lib/fathead/scala/parse.py

```diff
@@ -0,0 +1,75 @@
+from bs4 import BeautifulSoup
+import os
+
+BASE_DIRECTORY = "./download/api/scala"
+BASE_URL = "https://www.scala-lang.org/api/current/"
+
+
+def get_all_files():
+	doc_files = []
+
+	for (path, dirs, files) in os.walk(BASE_DIRECTORY):
```

Comment:
@moollaza I actually used only scala directory while developing and forgot to add other directories while making the pull 😄